jarpcbind/src/main/java/org/dcache/jarpcbind/Main.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2009 - 2012 Deutsches Elektronen-Synchroton,
```

Comment:
actually, I don't know the 'legal' rules. I just update header as soon as I touch the file.